modules/navier_stokes/src/actions/NSFVAction.C

```diff
@@ -2148,6 +2168,52 @@ NSFVAction::processVariables()
                      ") supplied to the NavierStokesFV action does not exist!");
 }
 
+bool
+NSFVAction::processThermalConductivity()
+{
+  checkBlockwiseConsistency<MooseFunctorName>("thermal_conductivity_blocks",
+                                              {"thermal_conductivity"});
+  bool have_scalar = false;
+  bool have_vector = false;
+
+  for (unsigned int i = 0; i < _thermal_conductivity_name.size(); ++i)
+  {
+    // First, check if the name is just a number (only in case of isotropic conduction
+    std::istringstream ss(_thermal_conductivity_name[i]);
+    Real real_value;
+    if (ss >> real_value && ss.eof())
+      have_scalar = true;
```

Comment:
we really ought to have a routine to do this

Reply:
Well, I looked up the parser routines (grepping `if (ss >> real_value && ss.eof())`)  but it looks like it is explicitly written out every time, couldn't find a routine which is dedicated for this alone. I added a routine because we use this multiple times in the action.